presto-main/src/main/java/com/facebook/presto/cost/JoinStatsRule.java

```diff
@@ -49,12 +49,21 @@
         implements ComposableStatsCalculator.Rule
 {
     private static final Pattern PATTERN = Pattern.typeOf(JoinNode.class);
+    private static final double DEFAULT_UNMATCHED_ANTI_JOIN_NDVS_COEFFICIENT = 0.5;
```

Comment:
q78 looks fine, and if there isn't a regression for the other queries either then this looks good.